app/views/tasks/show.html.erb

```diff
@@ -0,0 +1,25 @@
+<h2><%= @task.name %></h2>
+<p><%= @task.description %></p>
+<p>Priority:
+  <% if @task.priority == 1 %>
+    <span>High</span>
+  <% elsif @task.priority == 2 %>
+    <span>Mid</span>
+  <% elsif @task.priority == 3 %>
+    <span>Low</span>
+  <% end %>
+</p>
+<h4>
+  <% if @task.completed %>
+    Completion Date: <%= @task.completion_date %>
+  <% else %>
+    Not Completed
+  <% end %>
+</h4>
+
+<div class="show-options">
+  <% if @task.completed == false %>
```

Comment:
This isn't showing up because when you make the new task you're setting `completed` to `nil` instead of `false`.